components/cascader/style/index.less

```diff
@@ -23,6 +23,10 @@
     position: relative;
   }
 
+  &-show-search.ant-focusd {
```

Comment:
Thanks for your PR, we will automatically sync the antd style with [ant-design](https://github.com/ant-design/ant-design) (which means this change will be overwritten), if [ant-design](https://github.com/ant-design/ant-design) has the same problem, can you submit this PR to them? if not please write the styles in the `cascader/style/patch.less` file. ❤️

Reply:
@hsuanxyz 
I've reverted the change to `cascader/style/index.less` and synced the class name to be consistent with ant-design.

https://github.com/ant-design/ant-design/blob/3fcecdde0a54a0734af8e9d2287520db3a092a1c/components/cascader/index.tsx#L535

Pls take a look